README: update mention on tests, add docs for options (#14)
ExodusMovement/zip-up: README.md

// bytesWritten -> 276
```

## Options

Options can be passed as the second argument to `addDir`:

```js
await zipper.addDir('directory', {
  ignoreHidden: true,
  exclude: ['/tmp/']
})
```

`ignoreHidden`: whether to ignore hidden files (default: `false`)

> `exclude`: array of paths to exclude (default: `[]`)

Reviewer: Mention that it will also exclude files if they match the pattern.

Author: Done :+1:

_Note_: this applies to files as well as directories.

## Testing

> Run `npm test` to run a simple test suite; this tests that no errors occur and

Author: Not quite simple as it was, could possibly use a better adjective.

Reviewer: ```suggestion
Run `npm test` to run the test suite. The tests insure that no errors occur and
```
